knip.jsonc

```diff
@@ -0,0 +1,46 @@
+{
+    "workspaces": {
+        ".": {
+            // These entries are complementary to the ones found in package.json
+            "entry": [
+                "lib/rules/index.js",
+                "tools/internal-rules/index.js",
+                "tools/update-rule-types.js",
```

Comment:
[Question] Maybe `tools/update-rule-types.js` should be deleted? It's not referenced by anything and doesn't look like anything more than a one-time command.

_(probably a question for a team member)_

Reply:
Yeah, it was a one-time thing, just kept it around because we had to do several rounds before we got everything correct. It's probably safe to remove at this point.

Reply:
https://github.com/eslint/eslint/issues/18121